src/main/java/org/geomerty/procedural/shapes/Triangle.java

```diff
@@ -0,0 +1,6 @@
+package org.geomerty.procedural.shapes;
+
+public class Triangle {
```

Comment:
@LouiseLM There are several ways to define a triangle. e.g. Another way would be by specifying 2 adjacent sides and angle between them. No matter how you specify it, it's still a triangle. It becomes evident how encapsulation works in OOP version of the code.